lib/DDG/Spice/CodeSearch.pm

```diff
@@ -19,7 +20,7 @@ handle remainder => sub {
 
         s/\s*$match\s*//i;
 
-        return "lang:".$match." ".$_;
+        return $match." ".$_;
```

Comment:
is there no better way of specifying the language we want results for? when I played around with the API I was noticing results from several languages.

Do we have access to any API documentation?